src/app/services/weatherContainer.service.js

```diff
@@ -1,6 +1,8 @@
 export default class OpenWeatherMapsService {
-    constructor() {
-        this.nameDays = ['SUNDAY', 'MONDAY', 'TUESDAY', 'WEDNESDAY', 'THURSDAY', 'FRIDAY', 'SATURDAY'];
+    constructor($filter) {
+        this.nameDays = [$filter('translate')('CONTAINER_DAYS.NAME_DAYS.SUNDAY'), $filter('translate')('CONTAINER_DAYS.NAME_DAYS.MONDAY'), $filter('translate')('CONTAINER_DAYS.NAME_DAYS.TUESDAY'), $filter('translate')('CONTAINER_DAYS.NAME_DAYS.WEDNESDAY'), $filter('translate')('CONTAINER_DAYS.NAME_DAYS.THURSDAY'), $filter('translate')('CONTAINER_DAYS.NAME_DAYS.FRIDAY'), $filter('translate')('CONTAINER_DAYS.NAME_DAYS.SATURDAY')];
```

Comment:
Would be better to define `$filter('translate')` as a function and then reuse it, instead of get translate filter every time